Microsoft.Toolkit.Uwp.UI/Extensions/TextBoxRegEx/TextBoxRegex.Data.cs

```diff
@@ -17,7 +17,9 @@ namespace Microsoft.Toolkit.Uwp.UI.Extensions
     /// </summary>
     /// <remarks>
     /// If <see cref="ValidationMode"> is set to Normal then IsValid will be set according to whether the regex is valid.</see>
-    /// If <see cref="ValidationMode"> is set to Forced and the input is not valid the TextBox text will be cleared.</see>
+    /// If <see cref="ValidationMode"> is set to Forced, the input will be validated if the TextBox loses focus, if it is invalid, the TextBox text will be deleted.</see>
+    /// If <see cref="ValidationMode"> is set to Instantly, the input will be validated immediately, if it is invalid, the text of the TextBox will be deleted.</see>
```

Comment:
Is this useful?  If the user types something wrong it'll erase all their text, no?

I think the 'Dynamic' mode is more standard?

Also, have we checked how this behaves with Undo?

Reply:
@michael-hawker I agree, but I didn't want to change with this PR the extension, only expand. If it is wanted, I would delete force and instantly.

I debuged it and it works with undo and copy&paste.

Reply:
I agree that instantly seems unnecessary, I would not want to have to retype something just because i miss-typed one character.